services/slack/parse.go

```diff
@@ -285,7 +285,7 @@ func (t *Transformer) SlackConvertPostsMarkup(posts map[string][]SlackPost) map[
 			for _, rule := range regexReplaceAllStringFunc {
 				result = rule.regex.ReplaceAllStringFunc(result, rule.fn)
 			}
-			posts[channelName][postIdx].Text = result
+			posts[channelName][postIdx].Text = truncateRunes(result, model.PostMessageMaxRunesV2)
```

Comment:
There's a script I wrote that separates these into separate posts/replies https://github.com/mickmister/mmimportjs/blob/master/src/fix-posts.ts, but I think this is a better approach. There is a bit of data loss with truncating the post, but I think that's acceptable here

Reply:
Oh interesting! The one thing here is that some are actually replies, and breaking them up breaks a lot of the formatting is what I'm noticing. The really long ones in this example are just logs and such pasted though. 

If it makes sense to just make two posts with the same thread ID, I can do that too!